### PR TITLE
update babel plugin

### PR DESCRIPTION
**Pull request by JoviDeCroock**
https://astexplorer.net/#/gist/7098a93b18f9000dafa4a38ec985ee96/96b512473db6f9c4cebcf8a544b44cd98436efcc

### CODE REVIEW

File: scripts/babel/transform-debug-target.js [hunk 1]
```diff
@@ -16,15 +15,32 @@ const plugin = ({ template, types: t }) => {
 
   return {
     visitor: {
+      ObjectProperty(path) {
+        if (path.node.key && path.node.key.name === dispatchProperty && !path.node[visited]) {
+          path.node[visited] = true;
+		      path.node.value = t.conditionalExpression(
```
Comment (andyrichardson) [inline scripts/babel/transform-debug-target.js]: Just curious - how come you went from the template syntax to this?

Comment (JoviDeCroock) [reply scripts/babel/transform-debug-target.js]: templating will always add `semicolons` which isn't desirable for properties

Comment (kitten) [reply scripts/babel/transform-debug-target.js]: there's alternatives in `@babel/template` for this, but I'm fine with getting a working version in first :) https://babeljs.io/docs/en/babel-template#templateexpression

Comment (andyrichardson) [reply scripts/babel/transform-debug-target.js]: Removing the semicolon after NODE in the template string (`NODE;`) doesn't fix that?

Comment (JoviDeCroock) [reply scripts/babel/transform-debug-target.js]: My template doesn't have a semi https://github.com/FormidableLabs/urql/pull/671/commits/c4e657c025afbe6de6b527942dd7a188086d4357#diff-21f64d2a8eadde32a78a9f45d0587966L9

Comment (andyrichardson) [reply scripts/babel/transform-debug-target.js]: Woah, that's super unusual

**Pull request merged**